### ISSUE TITLE
SkyBlock Top Ten Menu Title

### ISSUE THREAD

**Issue by bertek41** [tastybento/askyblock#511]
/is top menu title can not be changed in locale.yml can you fix it?

**Comment by tastybento** [comment]
locale.yml is going away in the next version so this is moot. But you should check that you changed the title in the language file you use as well as locale.yml.

**Comment by tastybento** [comment]
Oh, I see what you mean - the GUI title string is missing from the locale fles. Just add it:

```
topTen:
  errorExcluded: "FYI: You are excluded from the top ten as you do not have [perm]"
  errorNotReady: "Top ten list not generated yet!"
  guiTitle: "Top 10 Islands"
  header: "These are the Top 10 islands:"
```

**Comment by tastybento** [comment]
Fixed by 	0478d36
